docs/setuptools.txt

```diff
@@ -569,43 +569,6 @@ on "entry points" in general, see the section below on `Dynamic Discovery of
 Services and Plugins`_.
 
 
-"Eggsecutable" Scripts
```

Comment:
As long as it's deprecated and not *removed*, I think we should put a deprecation notice on the documentation rather than removing the documentation.

Reply:
Done.